### ISSUE TITLE
I can I burn all editions of a piece of art?

### ISSUE THREAD

**Issue by ashwhite** [hicetnunc2000/hicetnunc#835]
I minted an OBJKT and created 100 editions of that artwork. I retained one edition with the remaining 99 available to buy. I now want to burn all editions, but I'm only able to burn the one edition I 'collected', not all remaining pieces.

How can I completely remove all editions of the artwork?

**Comment by andrevenancio** [comment]
Can you provide a little more information? What is your wallet ID or the OBJKT ID?

**Comment by ashwhite** [comment]
Hi there,

I think managed to work this out. I had to cancel the swap first before all
were redistributed back to me. Then I could burn them all. I just couldn’t
find any info regarding the need to go through that process.

Thanks.

**Comment by skenaja** [comment]
![image](https://user-images.githubusercontent.com/510285/121678468-22640580-caaf-11eb-81cd-f94ea0a07d9e.png)

This is mentioned on the burn page - did you spot that text? Do you have any suggestions for improving the wording or visibility of the message?

**Comment by ashwhite** [comment]
Thanks for sharing. I did read that but I guess I got confused with what
‘swapping’ actually means. The work talks of editions and collections so
‘ownership’ of editions wasn’t overly clear to me. Perhaps something a
little more black and white would work better? Or clearer differentiation
between what is owned and what is available to purchase.

“If you created an OBJKT with multiple editions but only ‘own’ (collected)
one edition and wish to burn all editions, including the ‘floating’
editions you’ve made available for purchase, you must do....”

I don’t know if that helps at all but maybe it was just me getting confused
on the terminology etc.

Thanks for your response.
